### ISSUE TITLE
Support for model2vec

### ISSUE THREAD

**Issue by do-me** [xenova/transformers.js#970]
### Feature request

Model2vec would make for a great addition to transformers.js: 
https://github.com/MinishLab/model2vec

### Motivation

The library distills conventional embedding models and produces static models that are magnitudes faster with a drop in accuracy of 10-20%. Also it got added to Sentence Transformers lately.

### Your contribution

See https://github.com/MinishLab/model2vec/issues/75#issuecomment-2407139446, there has been more requests to integrate it. One author said: 

> You are the second person to ask for this today, so let's do it. I haven't work with transformers.js at all, but we'll figure it out probably.

**Comment by xenova** [comment]
This should be possible! I think the simplest thing would be to match the normal feature-extraction model signature (input_ids + attention_mask, optionally).

**Comment by vtempest** [comment]
Yes it'd be greatly beneficial and faster than minilm v6. Could be the new default

**Comment by puppetm4st3r** [comment]
any news on this? regarda

**Comment by xenova** [comment]
model2vec models that are exported in the right way are compatible with transformers.js 🙌 See the thread for more information: https://github.com/MinishLab/model2vec/issues/75
